renderer/pages/[folder]/[path]/[file]/index.tsx

```diff
@@ -56,6 +62,16 @@ export const Files = () => {
     setEditing(false);
   };
 
+  const add = async () => {
+    const res = await fetch(`/api/hltb?game=${activeRom.info?.title}`);
+    const items: HLTGame[] = await res.json();
+
+    if (items.length > 0) {
+      await addToBacklog(items[0]);
+      router.push("/backlog");
```

Comment:
Same comments

Reply:
Also maybe add a toast saying game.name was added

Reply:
Ah! Is the intrusivity the navigating to the backlog?

We could replace that with the toast, what do you think? The toast is a great idea imo

Reply:
Yes sounds good!